Sustainsys.Saml2.AspNetCore2/Saml2Handler.cs

```diff
@@ -52,7 +52,7 @@ public Saml2Handler(
         /// <InheritDoc />
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Maintainability", "CA1500:VariableNamesShouldNotMatchFieldNames", MessageId = "scheme")]
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Maintainability", "CA1500:VariableNamesShouldNotMatchFieldNames", MessageId = "context")]
-        public Task InitializeAsync(AuthenticationScheme scheme, HttpContext context)
+        public virtual Task InitializeAsync(AuthenticationScheme scheme, HttpContext context)
```

Comment:
these methods were changed to virtual to allow overriding them for custom authentication implementation